fenixedu-ist-giaf-invoices/src/main/java/org/fenixedu/bennu/GiafInvoiceConfiguration.java

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright © 2013 Instituto Superior Técnico
+ *
+ * This file is part of FenixEdu IST GIAF Contracts.
+ *
+ * FenixEdu IST GIAF Contracts is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * FenixEdu IST GIAF Contracts is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with FenixEdu IST GIAF Contracts.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.fenixedu.bennu;
+
+import org.fenixedu.bennu.spring.BennuSpringModule;
+import org.fenixedu.commons.configuration.ConfigurationInvocationHandler;
+import org.fenixedu.commons.configuration.ConfigurationManager;
+import org.fenixedu.commons.configuration.ConfigurationProperty;
+
+@BennuSpringModule(basePackages = "pt.ist.fenixedu.giaf.invoices.ui", bundles = "GiafInvoicesResources")
+public class GiafInvoiceConfiguration {
+
+    @ConfigurationManager(description = "Giaf Invoice Configuration")
+    public interface ConfigurationProperties {
+
+        @ConfigurationProperty(key = "pt.indra.mygiaf.invoice.dir", defaultValue = "/dev/null",
+                description = "File store used to store files regarding invoices and reciepts from GIAF.")
+        public String giafInvoiceDir();
```

Comment:
1. Because I don't want transaction problems storing this stuff in the local system. If Giaf gives me an invoice I cannot lose it or else I can't retrieve it later.
2. Because this is a temporary solution. The entire payment system has to be replaced until the end of this year... so I tried to keep this as simple as possible.